Cargo.toml

```diff
@@ -199,3 +199,15 @@ tests_outside_test_module = "deny"
 unwrap_in_result = "deny"
 unwrap_used = "deny"
 use_debug = "deny"
+
+[profile.release]
+codegen-units = 1
```

Comment:
the compile times :P

Reply:
fwiw, this qualitatively didn't seem to make a real difference in compile times, because we have tons of parallelism in our crate dependency graph

Reply:
The default of 16 is not that good probably